README.md

```diff
@@ -73,8 +73,6 @@ This performs several optimizations which can significantly reduce the final siz
 
 ## License
 
-Copyright 2020 Anchor Protocol
+This repository is licensed under the Apache 2.0 license. See [LICENSE](./LICENSE) for full disclosure.
 
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
-See the License for the specific language governing permissions and limitations under the License.
+© 2020 Anchor Protocol.
```

Comment:
I rebased the branch to the recent main branch changes and fix the license issue.